examples/multidimensional_array.html

```diff
@@ -0,0 +1,176 @@
+<!--
+    Displaying the contents of a 3D Array 
+-->
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <script src="https://cdn.jsdelivr.net/npm/regular-table@0.0.6/dist/umd/regular-table.js"></script>
+    <link rel='stylesheet' href="https://cdn.jsdelivr.net/npm/regular-table@0.0.6/dist/css/material.css">
+
+    <style>
+        p {
+            display: block;
+            margin: 0;
+            height: 250px;
+            position: static;
+        }
+
+        td {
+            color: #1078d1;
+        }
+
+        tbody th:last-of-type,
+        thead tr:nth-child(2) th:nth-child(2),
+        thead tr:first-child th:first-child {
+            border-right: 1px solid #ddd;
+        }
+
+        regular-table {
+            height: 250px;
+            display: block;
+        }
+
+        regular-table div {
+            position: static;
+            height: auto;
+        }
+    </style>
+</head>
+
+<body>
+
+    <tables></tables>
+
+    <script>
+        // A random 3-dimensional data is created for
+        // implementing the regular-table concept for 3D array
+
+const DATA = [
```

Comment:
`DATA` here is too big, in that it's 60 lines of meaningless static data, and too small, in that 15 x 15 slices are not a good way to show off the capabilities of regular-table.

Part of the appeal of regular-table is virtualization, which means that it can easily be used to render extremely large data sets, such as in the two billion rows example: https://bl.ocks.org/texodus/483a42e7b877043714e18bea6872b039. We should do something more along those lines here